app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/DisplayDatasetActivityTest.kt

```diff
@@ -149,6 +149,7 @@ class DisplayDatasetActivityTest {
     fun clickOnMenuAddNewPictureWorks() {
         openActionBarOverflowOrOptionsMenu(getInstrumentation().targetContext)
         onView(withText(R.string.add_new_picture)).perform(click())
+        onView(withText("Use camera")).perform(click())

```

Comment:
I suggest using a resoruce as it above because it makes everything more flexible (e.g for translations etc)